llm_bench/python/convert.py

```diff
@@ -346,33 +345,99 @@ def convert_causal_lm(args):
         del pt_model
         gc.collect()
 
-    model = OVModelForCausalLM.from_pretrained(
-        args.model_id,
-        export=True,
-        compile=False,
-        trust_remote_code=True,
-        load_in_8bit=False,
-        config=AutoConfig.from_pretrained(args.model_id, trust_remote_code=True),
-    )
-    end = time.perf_counter()
+    if args.bettertransformer:
```

Comment:
I think for consolidation all conversion code in one place and avoid duplication, you can reuse https://github.com/openvinotoolkit/openvino.genai/blob/eccde42b3ea39802e01e050a7b0a7cec97254e19/llm_bench/python/convert.py#L243

it replicate original optimum flow with allowing configs overriding and patching models that are not part of optimum. All things model getting onnx config, conversion, saving configs, weights compression are already present on it, so it maybe simpler to integrate bettertransformers inside.